src/dsymbol/conversion/second.d

```diff
@@ -175,6 +175,16 @@ do
 
 	// Create symbols for the type suffixes such as array and
 	// associative array
+
+	// Build calltip, for the symbol
+	string callTip;
+	foreach (b; lookup.breadcrumbs[])
+	{
+		callTip ~= b;
```

Comment:
can you replace ARRAY_SYMBOL_NAME with `"[]"` here or would that add breaking changes?

AAs should probably be indicated by something more readable as well (possibly something like `[...]`, though not sure on what would be good here yet)

Reply:
We can't do that, because it could be a static array, so we first need to deduce what's inside

I thought about doing this, but i figured i'd go step by step to avoid too much changes to review

I'll try to do that in a separate PR, but not right now

Reply:
about static arrays: I think it might be a better idea to just have `[]` now, it's less wrong than what was there before and definitely more readable

Reply:
Good point, i'll try to implement that